int64 mesh_index
---
bool success
sensor_msgs/JointState planned_grasp_state